packages/react/src/components/Tile/next/Tile.js

```diff
@@ -499,6 +513,15 @@ export function ExpandableTile({
     setIsTilePadding(paddingTop + paddingBottom);
   }, []);
 
+  useIsomorphicEffect(() => {
+    if (getInteractiveContent(belowTheFold.current)) {
+      setInteractive(true);
+      return;
+    } else if (getInteractiveContent(aboveTheFold.current)) {
+      setInteractive(true);
+    }
```

Comment:
There's no null check inside of `getInteractiveContent`, so I think it might be best to check for current to not be `null`. What do you think?

```suggestion
    if (belowTheFold.current && getInteractiveContent(belowTheFold.current)) {
      setInteractive(true);
      return;
    } else if (aboveTheFold.current && getInteractiveContent(aboveTheFold.current)) {
      setInteractive(true);
    }
```

Reply:
@tay1orjones that's a good question, was thinking about this a little and I wonder if the convention for this could be:

- If it's a prop you don't own (like from `forwardRef`), check to see if it's defined
- If it's a prop you do own (like from `useRef`), then you would want it to error out since something has gone wrong
  - This is helpful because if the effect runs and the ref is null | undefined then it just will noop instead of telling you that something is wrong

Curious what you think

Reply:
@joshblack Yeah that's a good point. With this one being `useRef` we can leave it as is 👍